scripts/flags/flags.js

```diff
@@ -0,0 +1,341 @@
+'use strict';
+
+const babel = require('@babel/register');
+const {transformSync} = require('@babel/core');
+const Module = require('module');
+const path = require('path');
+const fs = require('fs');
+babel({
+  plugins: ['@babel/plugin-transform-modules-commonjs'],
+});
+
+const yargs = require('yargs');
+const argv = yargs
+  .parserConfiguration({
+    // Important: This option tells yargs to move all other options not
+    // specified here into the `_` key. We use this to send all of the
+    // Jest options that we don't use through to Jest (like --watch).
+    'unknown-options-as-args': true,
+  })
+  .wrap(yargs.terminalWidth())
+  .options({
+    csv: {
+      alias: 'c',
+      describe: 'output cvs.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    diff: {
+      alias: 'd',
+      describe: 'output diff of two or more flags.',
+      requiresArg: false,
+      type: 'array',
+      choices: [
+        'www',
+        'www-modern',
+        'rn',
+        'rn-fb',
+        'canary',
+        'next',
+        'experimental',
+        null,
+      ],
+      default: null,
+    },
+    sort: {
+      alias: 's',
+      describe: 'sort diff by one or more flags.',
+      requiresArg: false,
+      type: 'array',
+      default: 'next',
+      choices: [
+        'www',
+        'www-modern',
+        'rn',
+        'rn-fb',
+        'canary',
+        'next',
+        'experimental',
+      ],
+    },
+  }).argv;
+
+// Load ReactNativeFeatureFlags with __NEXT_MAJOR__ replace with 'next'.
+// We need to do string replace, since the __NEXT_MAJOR__ is assigned to __EXPERIMENTAL__.
+function getReactNativeFeatureFlagsMajor() {
```

Comment:
I had to update the script since `__NEXT_MAJOR__` is no longer a global. Since it's inline, we need to string replace the assignment in the module and virtually require the file.